ONBOARD.md

```diff
@@ -0,0 +1,183 @@
+This document describes how to add new CI jobs (utilizing ci-operator) for
+OpenShift components to the OpenShift CI system. In general, two steps are
+needed:
+
+1. [Prepare configuration file](#prepare-configuration-for-component-repo),
+   describing how ci-operator should build and test your component.
+2. [Add new jobs to Prow](#add-prow-jobs), hooking ci-operator to important
+   GitHub events for your repository.
+
+This walkthrough covers a simple use case of hooking up a job running source
+code level tests (unit, performance etc.) on pull requests on a component
+repository. Although ci-operator can perform more sophisticated test steps than
+simply running unit tests, these use cases are beyond the scope of this doc.
```

Comment:
Bruno will add this in so maybe just note that the doc is coming

Reply:
I would rather not - it's not that helpful to someone who actually needs more sophisticated info, and it should not be long until Bruno merges that doc. Once he does, I'll put a link to it here.